app/models/log.rb

```diff
@@ -1,8 +1,13 @@
 class Log < ApplicationRecord
+  after_initialize :set_uid
 
-  def write(trace=nil, &block)
+  def set_uid
```

Comment:
Instances of the old pattern:

app//controllers/export_controller.rb:36:    @log_uid = (Log.maximum(:uid) || 1) + 1
app//controllers/upload_controller.rb:20:    @last_job = Log.maximum(:uid) || 1